### PR TITLE
add an example jekyll site

### PR DESCRIPTION
**Pull request by RyanDraves**
Add files for making an example Jekyll site with `rules_ruby`. Also adds a CI job for testing that it builds.

### CODE REVIEW

**Comment by p0deje** [comment]
There are some failures on CI, I think macOS would be trivial to fix, while Windows looks more complicated. WOuld you mind taking a look?

**Comment by RyanDraves** [comment]
> There are some failures on CI, I think macOS would be trivial to fix, while Windows looks more complicated. WOuld you mind taking a look?

I believe I fixed up the Mac build. Do you mind if I remove Windows support for this example? I didn't find anything useful when searching for the error and have no Windows development experience.

**Comment by p0deje** [comment]
> I believe I fixed up the Mac build. Do you mind if I remove Windows support for this example? I didn't find anything useful when searching for the error and have no Windows development experience.

Yes, let's remove for now.

File: examples/jekyll/bazel-jekyll
Comment (p0deje) [inline examples/jekyll/bazel-jekyll:1]: Can you remove this?

**Review by p0deje — APPROVED** [review]
Looks great, I left one small comment about a symlink, otherwise will merge it.

File: examples/jekyll/bazel-jekyll
Comment (RyanDraves) [reply examples/jekyll/bazel-jekyll:1]: ah good catch

**Pull request merged**